doc/inverse.rst

```diff
@@ -65,12 +67,16 @@ Inverse Solutions
    make_lcmv
    apply_lcmv
    apply_lcmv_epochs
+   apply_lcmv_evoked_tfr
+   apply_lcmv_epochs_tfr
```

Comment:
Following `apply_inverse_tfr_epochs` (and e.g. `apply_inverse_psd_epochs`), this should be `apply_lcmv_tfr_epochs`.
`apply_lcmv_evoked_tfr` should probably become `apply_lcmv_tfr`, given that `apply_lcmv` also assumes evoked objects.

Same for DICS below.